.github/workflows/reusable-update-python-project-version.yml

```
@@ -48,3 +48,9 @@ jobs:
           github_token: ${{ steps.app-token.outputs.token }}
           changelog: true
 
+      - name: Release
```

```suggestion
      - name: Create GitHub release
```

Just to be clear it isn't a release to PyPI